install_config/install/docker_registry.adoc

```diff
@@ -95,11 +95,18 @@ Any images anyone has built or pushed into the registry would disappear.
 
 ==== Production Use
 
-For production use, you should attach a remote volume or
-link:../../install_config/persistent_storage/persistent_storage_nfs.html[use persistent storage] using
-`*PersistentVolume*` and `*PersistentVolumeClaim*` objects for storage for the
-registry. For example, to attach an existing NFS volume to the registry once it
-has been defined:
+For production use, attach a remote volume or
+link:../../install_config/persistent_storage/index.html[use the persistent
+storage method of your choice].
+
+For example, to use an existing persistent volume claim:
+----
+$ oc volume deploymentconfigs/docker-registry --add --name=v1 -t pvc \
+     --claim-name=<pvc_name> --overwrite
+----
+
+Or, to attach an existing NFS volume to the registry once it has been
```

Comment:
Think the original intent here was referring to the registry. Kinda strange to mention that here but not in the preceding PVC example. Maybe move the " to the registry once it has been defined" sentiment up before both examples, in the "For production use" intro sentence?